template projects/telegram bot/dc2db3f2-46a2-431e-a901-386118a5a17b.json

```diff
@@ -0,0 +1,230 @@
+{
+  "attributes": {
+    "root": true, 
+    "template": true
+  },
+  "description": "Basic telegram webhook with registration.\n\nThe webhook send back the message.\n\nPrecondition is a bot token.\nMore: https://core.telegram.org/bots/tutorial#obtain-your-bot-token",
+  "interface": {
+    "inputs": {},
+    "outputs": {}
+  },
+  "implementations": {
+    "cranq": {
+      "nodes": {
+        "8253ddd3-0709-42eb-b91e-309f5d3acf43": {
+          "coordinates": [
+            -691,
+            -8
+          ],
+          "description": "",
+          "params": {},
+          "prototype": "d227e22b-e827-4c9d-99db-f96da179bd52",
+          "name": "start"
+        },
+        "79688e50-8550-40b8-afea-3c7497fcb9f3": {
+          "coordinates": [
+            532,
+            287
+          ],
+          "description": "",
+          "params": {},
+          "prototype": "1d2a6438-e4bd-4105-b491-cebbb4cae348",
+          "name": "log"
+        },
+        "67b8d39e-a115-4d64-b705-c5ad1ac67ab6": {
+          "coordinates": [
+            -181.22500000000002,
+            248.375
+          ],
+          "name": "register webhook -  FILL IT!!!",
```

Comment:
It seems there are not "log missing webhook" functionality.
Only for bot params.
Is it intentional?